library/std/src/keyword_docs.rs

```diff
@@ -397,6 +403,7 @@ mod enum_keyword {}
 /// [Rust book]:
 /// ../book/ch19-01-unsafe-rust.html#using-extern-functions-to-call-external-code
 /// [Reference]: ../reference/items/external-blocks.html
+/// [`crate`]: keyword.crate.html
```

Comment:
This section was missing a resolution so it was linking to `std` instead of the `crate` keyword docs.